react/wbooks/src/app/screens/Home/index.js

```diff
@@ -9,14 +8,12 @@ import Home from './layout';
 class HomeContainer extends Component {
   state = { books: { books: [] } };
 
-  getBooks = () => getBooks(store.getState().auth.accessToken);
-
   componentDidMount() {
     store.subscribe(() => {
       const { books } = store.getState();
       this.setState({ books });
     });
-    return this.getBooks().then(books => store.dispatch(booksActionsCreators.addBooks(books)));
+    store.dispatch(booksActionsCreators.getBooks());
```

Comment:
at this point you should be using react-redux

Reply:
The card says that implementing react-redux is optional :P . In wolox hackaton I used it and also the other day Capi explained to me so I feel comfortable with it. I would prefer to continue with other things instead of doing that in the training.
But without any doubts when I have to develop a react project I will use react-redux!